### PR TITLE
PXC-4291 Release notes for Percona XtraDB Cluster 5.7.43

### PR DESCRIPTION
**Pull request by alina-derkach-oaza**
new file:   docs/release-notes/5.7.43-31.66.md
	modified:   mkdocs-base.yml
	modified:   variables.yml

### CODE REVIEW

File: docs/release-notes/5.7.43-31.65.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+Improvements and bug fixes provided by Oracle for MySQL 5.7.43 and included in Percona XtraDB Cluster are the following:
+
+* OpenSSL 1.1.1 library has been upgraded to OpenSSL 3.0.9.
+
+Find the full list of bug fixes and changes in the [MySQL 5.7.43 Release Notes](https://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-43.html).
+
+## Bug fixes
+
+* [PXC-4218](https://jira.percona.com/browse/PXC-4218): The output of the SHOW VARIABLES LIKE 'wsrep_provider_options' was truncated.
```
Comment (patrickbirch) [inline docs/release-notes/5.7.43-31.65.md]: output of the SHOW
remove "the" or 
the SHOW VARIABLES LIKE 'wsrep_provider_options' command

**Review by patrickbirch — APPROVED** [review]
one slight change otherwise good

File: docs/release-notes/5.7.43-31.65.md (hunk 1 above)
Comment (alina-derkach-oaza) [reply docs/release-notes/5.7.43-31.65.md]: updated

File: docs/release-notes/5.7.43-31.65.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+Improvements and bug fixes provided by Oracle for MySQL 5.7.43 and included in Percona XtraDB Cluster are the following:
+
+* OpenSSL 1.1.1 library has been upgraded to OpenSSL 3.0.9.
+
+Find the full list of bug fixes and changes in the [MySQL 5.7.43 Release Notes](https://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-43.html).
+
+## Bug fixes
+
+* [PXC-4218](https://jira.percona.com/browse/PXC-4218): The output of SHOW VARIABLES LIKE 'wsrep_provider_options' was truncated.
```
Comment (kamil-holubicki) [inline docs/release-notes/5.7.43-31.65.md]: I recommend saying explicitly that due to this bug fix, it is absolutely necessary to follow this upgrade procedure
https://www.percona.com/doc/percona-xtradb-cluster/5.7/howtos/upgrade_guide.html
especially point 7 is required:
`Open another session and run mysql_upgrade.`

(see https://jira.percona.com/browse/PXC-4271)

Comment (alina-derkach-oaza) [reply docs/release-notes/5.7.43-31.65.md]: updated

File: docs/release-notes/5.7.43-31.65.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# Percona XtraDB Cluster 5.7.43-31.65 (2023-08-31)
+
+Percona XtraDB Cluster (PXC) supports critical business applications in your public, private, or hybrid cloud environment. Our free, open source, enterprise-grade solution includes the high availability and the security features your business requires to meet your customer expectations and business goals.
+
+## Release highlights
+
+Due to the [PXC-4218](https://jira.percona.com/browse/PXC-4218) bug fix, you must follow the upgrade procedure in the [Upgrading Percona XtraDB Cluster](https://www.percona.com/doc/percona-xtradb-cluster/5.7/howtos/upgrade_guide.html) guide. The `Open another session and run mysql_upgrade` step is a must-have.
```
Comment (patrickbirch) [inline docs/release-notes/5.7.43-31.65.md]: The [PXC-4218](https://jira.percona.com/browse/PXC-4218) bug fix requires the upgrade procedure in the ... guide. Be sure that you open another session and run `mysql_upgrade` as part of the process.

Comment (alina-derkach-oaza) [reply docs/release-notes/5.7.43-31.65.md]: done

**Review by kamil-holubicki — APPROVED** [review]

**Pull request merged**